lib/floe/workflow/choice_rule/data.rb

```diff
@@ -75,34 +75,38 @@ def presence_check(context, input)
           !rhs
         end
 
-        def is_null?(value) # rubocop:disable Naming/PredicateName
-          value.nil?
+        # rubocop:disable Naming/PredicateName
+        # rubocop:disable Style/OptionalBooleanParameter
+        def is_null?(value, ret_value = true)
```

Comment:
If we come up with a better term for the test, I'm thinking the same term could be used here instead of `ret_value`, which is kind of generic.

Reply:
I like predicate as good as any.